src/tests/data/skeleton.edn

```diff
@@ -0,0 +1,38 @@
+{:filter  ["Linear", "Linear"]
+ :format  "RGBA8888"
+ :name    "skeleton.png"
+ :pma     false
+ :regions [{:bounds  [2, 42, 352, 785]
+            :index   -1
+            :name    "first animation test v1002"
+            :offsets [324, 162, 1000, 1149]
```

Comment:
It is converted from the spine format. I decided to include everything since it could also be used in authoring tools, including some that we will build later.

Reply:
Can we change the converter to only output what we use currently? (if it's not too much work)
People will probably use the tests as code samples, and there will be a bunch of stuff that does nothing